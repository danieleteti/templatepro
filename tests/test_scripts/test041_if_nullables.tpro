---
1
{{if dataitemnullableallnull }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if !dataitemnullableallnull }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullInt32 }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullInt32 }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullInt64 }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullInt64 }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullString }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullString }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullBoolean }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullBoolean }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullCurrency }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullCurrency }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullDate }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullDate }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullTime }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullTime }}
OK
{{else}}
WRONG
{{endif}}
4
---
1
{{if dataitemnullableallnull.NullDateTime }}
WRONG
{{else}}
OK
{{endif}}
4
---
1
{{if !dataitemnullableallnull.NullDateTime }}
OK
{{else}}
WRONG
{{endif}}
4
---