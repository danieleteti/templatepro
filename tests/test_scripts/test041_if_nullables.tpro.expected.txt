---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---
1
OK
4
---